src/main/java/com/minecolonies/coremod/entity/pathfinding/PathNavigate.java

```diff
@@ -336,7 +343,7 @@ protected void pathFollow()
             //  If current node is bottom of a ladder, then stay on this node until
             //  the entity reaches the bottom, otherwise they will try to head out early
             if (pEx.isOnLadder && pEx.ladderFacing == EnumFacing.DOWN
-                    && !pExNext.isOnLadder)
+                  && !pExNext.isOnLadder)
             {
                 final Vec3d vec3 = getEntityPosition();
                 if ((vec3.yCoord - (double) pEx.yCoord) < 0.001)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 0.001 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)